CMakeLists.txt

```diff
@@ -107,6 +107,9 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/library")
 option(FLB_ALL                 "Enable all features"                         No)
 option(FLB_DEBUG               "Build with debug mode (-g)"                 Yes)
 option(FLB_RELEASE             "Build with release mode (-O2 -g -DNDEBUG)"   No)
+if (CMAKE_SYSTEM_PROCESSOR MATCHES "^(arm64|aarch64)")
+  set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsigned-char")
+endif()
```

Comment:
Ah, I'd overlooked the place.
These lines should be located around there: https://github.com/fluent/fluent-bit/blob/master/CMakeLists.txt#L70-L79

Reply:
Sure, I can move it and resubmit. Thank you.